mucommander-translator/src/main/resources/dictionary_en.properties

```diff
@@ -608,6 +608,8 @@ text_editor.spaces_to_tabs = Replace spaces with tabs
 text_editor.syntax_style = $[text_viewer.syntax_style]
 text_editor.tabs_to_spaces = Replace tabs with spaces
 text_editor.tab_size = $[text_viewer.tab_size]
+text_editor.undo = Undo
+text_editor.redo = Redo
```

Comment:
it doesn't do any harm but it's not needed either, please remove

Reply:
PR updated